group_vars/matrix_servers

```diff
@@ -1760,6 +1763,31 @@ matrix_client_hydrogen_self_check_validate_certificates: "{{ false if matrix_ssl
 
 
 
+######################################################################
+#
+# matrix-client-cinny
+#
+######################################################################
+
+matrix_client_cinny_enabled: false
```

Comment:
Seems like we need to add this here to make Cinny work on non-`amd64` platforms via self-building:

```yaml
matrix_client_cinny_container_image_self_build: "{{ matrix_architecture != 'amd64' }}"
```